scripts/kruize_metrics.py

```diff
@@ -120,15 +120,15 @@
         "addPerformanceProfileToDB_max_success": "max(max_over_time(kruizeDB_max{{method=\"addPerformanceProfileToDB\",application=\"Kruize\",status=\"success\"}}[6h]))",
         "loadPerformanceProfileByName_max_success": "max(max_over_time(kruizeDB_max{{method=\"loadPerformanceProfileByName\",application=\"Kruize\",status=\"success\"}}[6h]))",
         "loadAllPerformanceProfiles_max_success": "max(max_over_time(kruizeDB_max{{method=\"loadAllPerformanceProfiles\",application=\"Kruize\",status=\"success\"}}[6h]))",
-        "kruizedb_memory": "(sum(container_memory_working_set_bytes{pod=~"'"postgres-deployment-[^-]*-[^-]*$"'",container=\"postgres\"}))",
-        "kruizedb_cpu_max": "max(sum(rate(container_cpu_usage_seconds_total{pod=~"'"postgres-deployment-[^-]*-[^-]*$"'",container=\"postgres\"}[6h])))",
+        "kruizedb_memory": "(sum(container_memory_working_set_bytes{pod=~"'"kruize-db-deployment-[^-]*-[^-]*$"'",container=\"kruize-db\"}))",
+        "kruizedb_cpu_max": "max(sum(rate(container_cpu_usage_seconds_total{pod=~"'"kruize-db-deployment-[^-]*-[^-]*$"'",container=\"kruize-db\"}[6h])))",
         "kruize_memory": "(sum(container_memory_working_set_bytes{pod=~"'"kruize-[^-]*-[^-]*$"'",container=\"kruize\"}))",
         "kruize_cpu_max": "max(sum(rate(container_cpu_usage_seconds_total{pod=~"'"kruize-[^-]*-[^-]*$"'",container=\"kruize\"}[6h])))",
         }
 
 def get_postgresql_metrics(namespace):
```

Comment:
Should we rename the method name as well ?

Reply:
Done